source/types/options.ts

```diff
@@ -99,7 +101,7 @@ export interface Options extends Omit<RequestInit, 'headers'> {
 
 	Accepts any value supported by [`URLSearchParams()`](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/URLSearchParams).
 	*/
-	searchParams?: string[][] | Record<string, string> | string | URLSearchParams;
+	searchParams?: SearchParamsInit | Record<string, string | number | boolean> | Array<Array<string | number | boolean>>;
```

Comment:
The whole type here should be exported as a named type `SearchParamsOption`.

Reply:
Done.